src/UISchema.tsx

```diff
@@ -0,0 +1,14 @@
+export interface IUISchema {
+  appBar: {
+    ["ui:title"]: string,
+    ["ui:logoUrl"]: string,
+    ["ui:inputPlaceholder"]: string,
+    ["ui:splitView"]: boolean,
+  };
```

Comment:
hmmm, `appBar` really isnt part of this repo. should rethink this

Reply:
@BelfordZ ^^ thoughts? should it be like [k: string]: any or something to allow `playground` to have another `IUISchema` that has `appBar` in it?